src/configuration.ts

```diff
@@ -112,14 +113,15 @@ export function findConfiguration(configFile: string | null, inputFilePath: stri
  * @returns An absolute path to a tslint.json file
  * or undefined if neither can be found.
  */
-export function findConfigurationPath(suppliedConfigFilePath: string | null, inputFilePath: string) {
+export function findConfigurationPath(suppliedConfigFilePath: string | null, inputFilePath: string): string | undefined;
+export function findConfigurationPath(suppliedConfigFilePath: string, inputFilePath?: string): string | undefined {
     if (suppliedConfigFilePath != null) {
         if (!fs.existsSync(suppliedConfigFilePath)) {
             throw new FatalError(`Could not find config file at: ${path.resolve(suppliedConfigFilePath)}`);
         } else {
             return path.resolve(suppliedConfigFilePath);
         }
-    } else {
+    } else if (inputFilePath !== undefined) {
```

Comment:
On second thought, you don't need this condition. If `suppliedConfigPath === null`, then `inputFilePath` cannot be undefined. You can simply assert that using the nonNullAssertion operator `inputFilePath!` in the else branch